components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/listener/OrganizationManagerListener.java

```diff
@@ -47,7 +46,7 @@ void postGetOrganization(String organizationId, Organization organization)
     default void postDeleteOrganization(String organizationId, int organizationDepthInHierarchy)
             throws OrganizationManagementException {
 
-        throw new NotImplementedException("This method is not implemented.");
+        // This method is added to avoid breaking changes. This method will be removed in the next major release.
```

Comment:
`This method will be removed in the next major release.` what this means? we will use this method in the future as well.
 The @Deprecated postDeleteOrganization will be removed in the next major release

Reply:
fixed

Reply:
should we keep the default function in the next release as well?